analyses/snv-callers/README.md

```diff
@@ -223,144 +85,65 @@ Where genome size is calculated from the respective BED file as:
 genome_size = sum(End_Position - Start_Position)
 ```
 
-BED regions for WXS samples can be [found here](https://raw.githubusercontent.com/AstraZeneca-NGS/reference_data/master/hg38/bed/Exome-AZ_V2.bed).
-BED regions used for WGS samples are caller specific are from <unknown as of now>
-The sample-wise TMB calculations written to a file ending in `_tmb` in the caller's results folder.
-
-*Output for this analysis*
-* `results/<caller_name>/<caller_name>_tmb.rds`
-* `plots/<caller_name>/<caller_name>_<strategy>_tmb_plot.png`
-
-### COSMIC Mutation Overlap
-
-The COSMIC mutation data were obtained from https://cancer.sanger.ac.uk/cosmic/download
-*To run this analysis, you need to obtain these data.*
-The full, unfiltered somatic mutations file `CosmicMutantExport.tsv` for grch38 is used here and the genomic coordinates is arranged to be in BED format.
-The COSMIC set is filtered down to only mutations detected in brain-related
-samples using the `Site subtype 1` field.
-COSMIC mutations are overlapped with the present data's mutations using `GenomicRanges`.
-The outcome of this overlap is added to the VAF data.frame with two `TRUE/FALSE` columns:
-`overlap_w_cosmic` is TRUE for mutations that overlap with COSMIC mutations, while `same_as_cosmic` is TRUE when the base change summary is also identical.
-The VAF for mutations that are or are not overlapping with COSMIC mutations are then plotted in a violin plot.
-
-*Output for this analysis*
-* `results/<caller_name>/<caller_name>_vaf.rds`
-* `plots/<caller_name>/<caller_name>_<strategy>_cosmic_plot.png`
+## General usage of scripts
 
-## Comparison of Callers
+**Overall notes about these scripts:**
+- The scripts are sequential as noted by their number.
+- All file path-related options assume the file path given is relative to `OpenPBTA-analysis`.
+- By default, the scripts will not overwrite existing files of the same name. However,
+this can be overridden with `--overwrite` option.
 
-After running an initial evaluation and set up of each of the callers' MAF files,
-the `compare_snv_callers.Rmd` notebook can be run to get final results on consensus
-mutation calls.  
+### 01-setup_db.py
 
-### Mutation IDs  
+Creates and/or fills a database of variant calls that will be used by subsequent calls.
```

Comment:
Probably worth mentioning that the maf tables are complete for strelka and lancet, but not for mutect and vardict, to save space. Here you might mention the indexing by position and mutation, to facilitate later searches and comparisons.
```suggestion
Creates and/or fills an SQLite database of variant calls that will be used by subsequent steps to find consensus mutations.
```